src/DIRAC/Resources/Computing/CloudComputingElement.py

```diff
@@ -283,6 +288,35 @@ def _getFlavor(self):
                 return flavor
         raise KeyError("No matching flavor found for %s" % rawID)
 
+    def _getNetworks(self):
+        """Extracts network list from configuration system.
+
+        :return: List of network objects or None if not set
+        """
+        rawIDs = self.ceParameters.get("Instance_Networks", None)
```

Comment:
Should this parameter be documented?

Reply:
Yes, we included a brief description with the rest of the documentation at the top of the file (from line 101).